src/components/common/ProfilePanel.js

```diff
@@ -316,6 +316,19 @@ class ProfilePanel extends React.Component {
     </div>
   );
 
+  renderShowStatusBar = () => {
+    let onToggle = () => {
+      console.log(this.props);
+      this.props.setStatusBar(!this.props.statusBar);
+    };
+    return (
+      <div className="panel-button">
+        <button className="statusbar-button" onClick={() => onToggle()}>
+          <span> Status Bar</span>
+        </button>
+      </div>
+    );
+  };
```

Comment:
For now, let's mask the button from the editor, then we can add a button for it when we create the collaborative editor. Apologies if that wasn't clear. As a side note, using the same classes as `themeSwitch`, etc. would benefit its uniformity:

![image](https://user-images.githubusercontent.com/7704542/108637639-5bb13080-7440-11eb-98b9-f52ed92935a0.png)